mobile/lib/ui/viewer/actions/file_selection_actions_widget.dart

```diff
@@ -602,6 +606,42 @@ class _FileSelectionActionsWidgetState
     }
   }
 
+  Future<String> saveImage(Uint8List bytes) async {
+    String path = "";
+    try {
+      final Directory root = await getTemporaryDirectory();
+      final String directoryPath = '${root.path}/enteTempFiles';
+      // Create the directory if it doesn't exist
+      final DateTime timeStamp = DateTime.now();
+      await Directory(directoryPath).create(recursive: true);
+      final String filePath = '$directoryPath/$timeStamp.jpg';
+      final file = await File(filePath).writeAsBytes(bytes!);
+      path = file.path;
+    } catch (e) {
+      debugPrint(e.toString());
+    }
+    return path;
+  }
+
+  ValueNotifier<String?> generatedPathNotifier = ValueNotifier<String?>(null);
+//Future function to go to next page
+  Future<String?> _nextPageForTesting(List<EnteFile> tempfile) async {
+    final Widget imageWidget = ShowImagePreviewFromTap(
+      tempEnteFile: tempfile,
+    );
+    await Future.delayed(const Duration(milliseconds: 100));
+    final double pixelRatio = MediaQuery.of(context).devicePixelRatio;
+    final bytesOfImageToWidget = await screenshotController.captureFromWidget(
+      imageWidget,
+      pixelRatio: pixelRatio,
+      targetSize: MediaQuery.of(context).size,
```

Comment:
Use `MediaQuery.sizeOf(context)` here. It reduces widget rebuilds in some cases. Don't know if it'll make any difference here, but it's safer to use it instead.

ref: https://github.com/flutter/flutter/pull/114459